shared/ClrFacade/TestCases.cs

```diff
@@ -689,6 +689,35 @@ public static ITestGenericInterface<string>[] CreateTestArrayGenericInterface()
          result[i] = new ImplITestGenericInterface();
       return result;
    }
+   
+   public static object IsNull(object obj)
+   {
+      return (obj == null); 
+   }
+
+   public static object IsNA(object obj)
+   {
+      return obj;
+   }
+
+   public static object IsNaN(object obj)
+   {
+      if ((double?)obj == null) 
+         return false;
+
+      return double.IsNaN((double)obj);
+   }
```

Comment:
Can you rewrite this with safe-cast style. 

like this
```
      if (!(obj is double d)) 
         return false;

      return double.IsNaN(d);
```

This safe-cast style makes it easier to read.